run.go

```diff
@@ -92,6 +92,11 @@ func run(context *cli.Context) error {
 		if err != nil {
 			return fmt.Errorf("Process state %s: %s", ps.String(), err)
 		}
+
+		// delete container's resources
+		if err := delete(pod.GetAllContainers()[0].ID(), true); err != nil {
```

Comment:
Thanks for the change, but could you do the factorization too please. And the comment from @dlespiau makes sense, it is always better to check we didn't get `nil` before to use a method from this instance.